java/spring/src/main/java/com/yugabyte/springdemo/controller/OrderController.java

```diff
@@ -50,22 +50,22 @@ public Order createOrder(@Valid @RequestBody Order order) {
     	Order newOrder = orderRepository.save(order);
     	
     	for (Order.Product orderProduct : order.getProducts()) {
-    		orderLineRepository.save(new OrderLine(newOrder.getOrderId(), orderProduct.getProductId()));
+    		orderLineRepository.save(new OrderLine(newOrder.getOrderId(), orderProduct.getProductId(), orderProduct.getUnits()));
```

Comment:
We seem to have some files that are not in the c# directory, lets not add unrelated fixes into one diff. Makes it harder to review. Better to put these into a separate diff with a description of what we are trying to accomplish.